index.html

```diff
@@ -328,7 +328,58 @@ <h3>CONOPY <span>&bull; Back End Dev &bull; 2015</span></h3>
               </div>
             </details>
           </p>
-        </artcle></section
-    ></a>
+        </artcle>
+        <section class="form-wrap-level-one">
+          <div class="form-wrap-level-two">
+            <div class="form-container">
+              <h2 class="form-title">Contact me</h2>
+              <p class="form-description">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I’d
+                love to help with it!
+              </p>
+              <form
+                action="https://formspree.io/f/mqkojebp"
+                method="post"
+                class="contact-form"
+              >
+                <ul>
+                  <li class="name-field">
+                    <input
+                      type="text"
+                      name="user-name"
+                      placeholder="Full name"
+                      maxlength="30"
+                      required
+                    />
+                  </li>
+                  <li class="email-field">
+                    <input
+                      type="email"
+                      name="user-email"
+                      placeholder="Email address"
+                      required
+                    />
```

Comment:
Kudos for the great effort you have made in your current milestone. However, you still have a few things to change
- Please consider fixing the position of the text in your text area input. It's currently almost in the middle which is different from the Figma design. Please take a look at the picture below for a better understanding.
- The li tag used for each input isn't ideal. Kindly remove it so your code can follow the proper semantics
- Currently, whenever I click any of the input fields, it redirects me to the beginning of your page which isn't a good user experience. Kindly check this section to make the necessary adjustments.
- Also, your form submission isn't successful at the moment.  Kindly check it out and make the necessary adjustments

![image](https://github.com/WilfriedM01/Portfolio-project/assets/89091822/03d1f914-4fe6-470d-9faf-690c2b6d0b9f)

Reply:
Hi @Eternalgratis 

Thank you for reviewing, I improve it immediately

Reply:
Hi @Eternalgratis 

I have improved the code and the User Experience. 
However, I choose to use li tag because It is used in the materials proposed in the course, and personally, it is making sense for me and make that one easier to style.

Thank you :+1: :v: